modules/op-installing-pipelines-operator-using-the-cli.adoc

```diff
@@ -27,7 +27,7 @@ spec:
   source: redhat-operators <3>
```

Comment:
from:
`name: openshift-pipelines-operator-rh <2>`
to:
`name: openshift-pipelines-operator-rh # <2>`